### PR TITLE
Adding hint tag to the labels, deduping tags.

### LINKED ISSUE #52: Handle Hint.Tag field
Currently this information is discarded, need to capture it and figure out where it belongs in the analyzer-lsp

### PR DESCRIPTION
**Pull request by shawn-hurley**
fixes #52

### CODE REVIEW

**Comment by shawn-hurley** [comment]
This has the same pass rate as 6.2.4, didn't not effect it one way or there other, I think it should be fine to merge this, and using the other PR to set the version we use.

**Comment by shawn-hurley** [comment]
Overall success rate: 81.09% (1917/2364)

@fabianvf @pranavgaikwad Can we merge these together?

File: pkg/conversion/convert.go [hunk 1]
```diff
@@ -587,6 +591,9 @@ func convertWindupPerformToAnalyzer(perform windup.Iteration, where map[string]s
 			ret["effort"] = i
 		}
 
+		if len(hint.Tag) != 0 {
```
Comment (pranavgaikwad) [inline pkg/conversion/convert.go:594]: isnt `hint.Tag` really an application tag?

**Review by pranavgaikwad — CHANGES_REQUESTED** [review]

File: pkg/conversion/convert.go (hunk 1 above)
Comment (shawn-hurley) [reply pkg/conversion/convert.go:594]: What else would it be?

Comment (shawn-hurley) [reply pkg/conversion/convert.go:594]: It is a particular tag, for a particular violation/incident or a rule, I can only assume that we want to map this to a rule label, right ?

**Review by pranavgaikwad — APPROVED** [review]

**Pull request merged**